zeppelin-web/src/app/home/home.html

```diff
@@ -12,6 +12,22 @@
 limitations under the License.
 -->
 
+<script type="text/ng-template" id="notebook_folder_renderer.html">
+  <div ng-if="node.children === undefined">
+    <i style="font-size: 10px;" class="icon-doc"></i>
+    <a style="text-decoration: none;" href="#/notebook/{{node.id}}">{{node.name || 'Note ' + node.id}}</a>
```

Comment:
`<i style="font-size: 10px;" class="icon-doc"></i>` should be within `<a>` tag. 
In this way, this gets the color of the text (instead of black), as well as, user can click on it. Like the way its used everywhere else in the page.